core/drivers/clk/sam/phy-sama7-utmi-clk.c

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Driver for the Microchip SAMA7 USB 2.0 PHY Clock
+ *
+ * Copyright (C) 2024 Microchip Technology, Inc. and its subsidiaries
+ *
+ * Author: Tony Han <tony.han@microchip.com>
+ *
+ */
+
+#include <drivers/atmel_rstc.h>
+#include <io.h>
+#include <kernel/delay.h>
+#include <mm/core_memprot.h>
+#include <sam_sfr.h>
+#include "at91_clk.h"
+
+#define SAMA7_SFR_UTMI0R(x) (0x2040 + 4 * (x)) /* offset of SFR_UTMI0Rx */
+#define SAMA7_SFR_UTMI_COMMONON BIT(3) /* PLL Common ON bit */
+
+struct sama7_utmi_clk {
+	struct clk hw;
```

Comment:
`hw` seems not being used.

Reply:
Yes, `hw` not used, the definition removed.